stanford_media.module

```diff
@@ -197,8 +197,8 @@ function stanford_media_theme_registry_alter(&$theme_registry) {
   // Register the path to the template file and add an additional variable to
   // image_formatter for link attributes.
   $path = drupal_get_path('module', 'stanford_media') . '/templates';
+  $theme_registry['responsive_image_formatter']['path'] = $path;
```

Comment:
This is assuming a new dependency on the responsive image module? Does it need to be enabled or added to the dependencies?

Reply:
https://github.com/SU-SWS/stanford_media/blob/8.x-1.x/stanford_media.info.yml#L26